torch_np/_ndarray.py

```diff
@@ -343,67 +399,80 @@ def __setitem__(self, index, value):
         return self._tensor.__setitem__(index, value)
 
 
-def asarray(a, dtype=None, order=None, *, like=None):
-    _util.subok_not_ok(like)
-    if order is not None:
+# This is the ideally the only place which talks to ndarray directly.
+# The rest goes through asarray (preferred) or array.
+
+def array(object, dtype=None, *, copy=True, order='K', subok=False, ndmin=0,
```

Comment:
nit. `object` is a builtin class in Python, perhaps use a different name?

Reply:
Yeah. So far I matched numpy names one to one, but this one is indeed unfortunate.